routes/routes.js

```diff
@@ -8,4 +23,10 @@ router.post('/palindrome', (req, res) => {
     //Palindrome logic goes here
 })
 
+router.post('/prime/:num', (req, res) => {
+    var num=req.params.num;
```

Comment:
Assume the num variable to be inside the body of the request. Please make the necessary change for this.